misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/reporting/TextExplanatoryAssertionGroupFormatterSpec.kt

```diff
@@ -5,16 +5,18 @@ import ch.tutteli.atrium.assertions.ExplanatoryAssertionGroupType
 import ch.tutteli.atrium.domain.builders.ExpectImpl
 import ch.tutteli.atrium.reporting.AssertionFormatter
 import ch.tutteli.atrium.reporting.AssertionFormatterController
+import org.spekframework.spek2.Spek
 import kotlin.reflect.KClass
 
 //TODO #116 migrate spek1 to spek2 - move to specs-common
 abstract class TextExplanatoryAssertionGroupFormatterSpec(
     testeeFactory: (Map<KClass<out BulletPointIdentifier>, String>, AssertionFormatterController) -> AssertionFormatter,
     describePrefix: String = "[Atrium] "
-) : TextExplanatoryBasedAssertionGroupFormatterSpec<ExplanatoryAssertionGroupType>(
+) : Spek({
+    include(object : TextExplanatoryBasedAssertionGroupFormatterSpec<ExplanatoryAssertionGroupType>(
```

Comment:
what's the reason that you have changed it to an include?  there is no additional spec going on in this class so IMO it is enough if it just extends TextExplanatoryBasedAssertionGroupFormatterSpec

Reply:
indeed it was not necessary (was required during development), I reverted back to original version